backend/endpoints/messages.py

```diff
@@ -77,3 +78,92 @@ async def send_message(
         status_code=status.HTTP_424_FAILED_DEPENDENCY,
         detail={"Message not sent": response},
     )
+
+
+@router.put(
+    "/org/{org_id}/rooms/{room_id}/messages/{message_id}",
+    response_model=ResponseModel,
+    responses={
+        status.HTTP_200_OK: {"description": "message edited"},
+        status.HTTP_404_NOT_FOUND: {"description": "message not found"},
+        status.HTTP_403_FORBIDDEN: {
+            "description": "you are not authorized to edit this message"
+        },
+        status.HTTP_424_FAILED_DEPENDENCY: {"description": "message not edited"},
+    },
+)
+async def update_message(
+    request: MessageUpdate,
+    org_id: str,
+    room_id: str,
+    message_id: str,
+    background_tasks: BackgroundTasks,
+):
+    """
+    Update a message
+
+    Args:
+        request: Request object
+        org_id: A unique identifier of the organization.
+        room_id: A unique identifier of the room.
+        sender_id: A unique identifier of the sender.
+        message_id: A unique identifier of the message that is being edited.
+
+    Returns:
+        HTTP_200_OK {message updated successfully}:
+        A dict containing data about the message that was updated (response_output).
+            {
+                "room_id": "619e28c31a5f54782939d59a",
+                "message_id": "61bc5e5378fb01b18fac1426",
+                "sender_id": "619ba4671a5f54782939d385",
+                "text": "testing edits",
+                "edited_at": "2021-12-17 11:47:22.678046"
+            }
+
+    Raises:
+        HTTP_404_FAILED_DEPENDENCY: Message not found
```

Comment:
I guess this must be a typo 'HTTP_404_FAILED_DEPENDENCY'

Reply:
oh yeah it is. Will fix